### PR TITLE
support get worker id info from config

### PR DESCRIPTION
**Pull request by dmthuc**
I hereby agree to the terms of the CLA available at: [placeholder]

Changelog category (leave one):
- New Feature
- Bug Fix
- Improvement
- Performance Improvement
- Backward Incompatible Change
- Build/Testing/Packaging Improvement
- Documentation (changelog entry is not required)
- Other
- Not for changelog (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

...

Detailed description / Documentation draft:

...

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ByConity Documentation](https://github.com/ByConity/ByConity/blob/master/CONTRIBUTING.md) guide first.

### CODE REVIEW

File: src/Common/HostWithPorts.cpp [hunk 1]
```diff
@@ -179,4 +180,34 @@ std::ostream & operator<<(std::ostream & os, const HostWithPorts & host_ports)
...
+{
+    char * ret = std::getenv(name.c_str());
+    if (ret)
+        return ret;
+
+    return context->getConfigRef().getString(name, "");
+}
+} /// end namespace
+
+std::string getWorkerId(ContextPtr context)
```
Comment (zeromem) [inline src/Common/HostWithPorts.cpp:195]: should be getWorkerID ?

File: src/Interpreters/CnchQueryMetrics/QueryMetricLogHelper.cpp [hunk 2]
```diff
@@ -337,7 +338,7 @@ void insertCnchQueryMetric(
     {
         String initial_query_id = context->getClientInfo().initial_query_id;
         String current_query_id = context->getCurrentQueryId();
-        String worker_id = getenv("WORKER_ID") ? getenv("WORKER_ID") : context->getHostWithPorts().getTCPAddress();
+        String worker_id = getWorkerGroupID(context);
```
Comment (zeromem) [inline src/Interpreters/CnchQueryMetrics/QueryMetricLogHelper.cpp:341]: should be getWorkerId ?

Comment (dmthuc) [reply src/Interpreters/CnchQueryMetrics/QueryMetricLogHelper.cpp]: Yeah, I've updated it

File: src/Common/HostWithPorts.cpp (hunk 1 above)
Comment (dmthuc) [reply src/Common/HostWithPorts.cpp]: Yeah, I've updated it

**Review by zeromem — APPROVED** [review]

**Pull request merged**